src/components/App.tsx

```diff
@@ -1305,6 +1310,11 @@ class App extends React.Component<ExcalidrawProps, AppState> {
   // Input handling
 
   private onKeyDown = withBatchedUpdates((event: KeyboardEvent) => {
+    // Allow copy only when inside canvas
+    const isCopyEvent = event[KEYS.CTRL_OR_CMD] && event.code === CODES.C;
+    if (isCopyEvent && this.lastMouseDownTarget !== this.canvas) {
+      return;
+    }
```

Comment:
Do we handle the `cmd+c` in any way outside of `onCopy`? If not, we should automatically return on `if (isCopyEvent)` here.

Also, can we use `event.key` here?